Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java

```diff
@@ -680,11 +680,8 @@ public void banFormatChange() {
     }
 
     private void updateBan(final String playerName, final String banReason, final Long banTimeout) {
-        if (banTimeout == 0) {
-            Bukkit.getBanList(BanList.Type.NAME).addBan(playerName, banReason, null, Console.NAME);
-        } else {
-            Bukkit.getBanList(BanList.Type.NAME).addBan(playerName, banReason, new Date(banTimeout), Console.NAME);
-        }
+        final Date date = banTimeout == 0 ? null : new Date(banTimeout);
+        Bukkit.getBanList(BanList.Type.NAME).addBan(playerName, banReason, date, Console.NAME);
```

Comment:
```suggestion
        Bukkit.getBanList(BanList.Type.NAME).addBan(playerName, banReason, banTimeout == 0 ? null : new Date(banTimeout), Console.NAME);
```

Reply:
@JRoy what's wrong with inline variables?

Reply:
You don't need to declare another variable here when you're just going to declare and use it once

Reply:
Ah, nvm, I see.
I've extracted it coz the line was too long and it's easier to read. But whatevs.